### ISSUE TITLE
unsupported conversion from 'Encoding: UTF-8' to '' ()

### ISSUE THREAD

**Issue by russHyde** [jimhester/lintr#404]
During CI for one of my packages, the installation of lintr from github failed (and I think it results from a change I made to lintr, sorry ...):

```
The downloaded source packages are in
	‘/tmp/RtmpZ2iKm8/downloaded_packages’
Installing R packages from GitHub: jimhester/lintr
4.79s$ Rscript -e 'remotes::install_github(c("jimhester/lintr"))'
Downloading GitHub repo jimhester/lintr@master
Running `R CMD build`...
* checking for file ‘/tmp/RtmpWaGmhe/remotes290663b7076c/jimhester-lintr-28a3765/DESCRIPTION’ ... EXISTS but not correct format
STDOUT:
* checking for file ‘/tmp/RtmpWaGmhe/remotes290663b7076c/jimhester-lintr-28a3765/DESCRIPTION’ ... EXISTS but not correct format
STDERR:
Error in iconv(out, encoding, "", sub = "byte") : 
  unsupported conversion from 'Encoding: UTF-8' to ''
Error: Failed to install 'lintr' from GitHub:
  Failed to `R CMD build` package, try `build = FALSE`.
Execution halted
```

The full CI notes are here: https://travis-ci.org/russHyde/miiq/builds/580344603?utm_source=github_status&utm_medium=notification

While reviewing / modifying #236 during https://github.com/jimhester/lintr/pull/398 , an "Encoding: UTF-8" field was added to the `DESCRIPTION` file. This was required to get the roxygen2  docs to compile. 

I'm not sure why the DESCRIPTION file is invalid and will try to find out from the 'writing R extensions' book.

**Comment by richelbilderbeek** [comment]
Same [here](https://travis-ci.org/richelbilderbeek/razzo/builds/580574702#L1375), actually in all my R repos. Good luck fixing it :+1:

**Comment by infotroph** [comment]
I think the issue is that there are now two Encoding fields. If #405 passes tests, I think it should resolve it.

**Comment by russHyde** [comment]
closed in #405
